modules/codegen/src/smithy4s/codegen/CollisionAvoidance.scala

```diff
@@ -244,7 +244,8 @@ object CollisionAvoidance {
     "BigDecimal",
     "Map",
     "List",
-    "Set"
+    "Set",
+    "None"
```

Comment:
Idk if this is the right solution here.. but it works. There was a collision between scala's Option#None and an enum member in the aws-http module

Reply:
yeah that's fine